lib/logger.js

```diff
@@ -1,3 +1,5 @@
+/// <reference path="../typings/node/node.d.ts" />
```

Comment:
also this is a .js file, so the comment does nothing? unless you're using TS 1.8 support
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#including-js-files-with---allowjs